spec/support/bulkrax/csv_writer_helper.rb

```diff
@@ -75,7 +75,14 @@ def fake_row_data(i)
   def faked_attribute(name, i)
     return unless name.is_a? Symbol
 
-    if attributes[name][:type] == "date"
+    if name == :creator_profile_visibility
```

Comment:
I do have one small suggestion for this. To try and avoid a scenario where by each time a field like this is added, this method needs to be adjusted, we could add something like... 

```ruby 
  def faked_attribute(name, i)
    return unless name.is_a? Symbol
    
    method_name = "fake_#{name}"
    if respond_to?(method_name)
        send(method_name)
    elsif attributes[name][:type] == "date"
    # ...
```

This way we have specifialist methods to deal with difficult fields/keys rather than an increasingly long if/case statement here. 

What do you think?

Reply:
I think that we should not have any special keys or exceptions here.  This is a temporary patch so that main passes again.  I think this list should only contain attribute types.  Had this test been in place when the creator_profile_visibility PR was being worked on it would have pointed to the fact that could have been done differently.